### PR TITLE
Edited plan_vmm_int.md

### PR DESCRIPTION
**Pull request by philip-menzies**
docs\get_started\plan_deployment\traditional_deployment\user_registry_consideration

### CODE REVIEW

File: docs/get_started/plan_deployment/traditional_deployment/user_registry_consideration/plan_ureg_ov.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # User registry options
 
-HCL Digital Experience provides various security configuration tasks. In the past, there was one task and you might not recover from errors. Also, you might not expand your user registry to meet your growing business needs. Now there are multiple tasks and you can fine-tune your system to meet your business needs.
+HCL Digital Experience provides various security configuration tasks. In previous versions, one task was available and error recovery was uncertain. Also in previous version, you were unable to expand your user registry to meet growing business needs. HCL DX 9.5 and later offer multiple tasks and you can fine-tune your system to meet your business needs.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/traditional_deployment/user_registry_consideration/plan_ureg_ov.md:3]:
```suggestion
HCL Digital Experience provides various security configuration tasks. In previous versions, one task was available and error recovery was uncertain. Also in previous versions, you were unable to expand your user registry to meet growing business needs. HCL DX 9.5 and later offer multiple tasks and you can fine-tune your system to meet your business needs.
```

File: docs/get_started/plan_deployment/traditional_deployment/user_registry_consideration/plan_ureg_ov.md [hunk 2]
```diff
@@ -17,23 +17,23 @@ HCL Portal is configured with a default federated repository with a built-in fil
...
 |----|-----------|
 |Updating the federated LDAP user registry|Choose this option to update certain parameters such as your bind ID and password to fix issues with your LDAP user registry.|
 |Updating the federated database user registry|Choose this option to update certain parameters such as the data source name, database URL, and database type to fix issues with your database user registry.|
-|Create a realm|Choose this option to create a realm, which is a group of users from one or more user registries that form a coherent group within HCL Portal. Realms allow flexible user management with various configuration options. A realm must be mapped to a Virtual Portal to allow the defined users to log in to the Virtual Portal. In a federated repository, you can create multiple realms.|
+|Create a realm|Choose this option to create a realm, which is a group of users from one or more user registries that form a coherent group within HCL Portal. Realms facilitates flexible user management with various configuration options. A realm must be mapped to a virtual portal to allow the defined users to log in to the virtual portal. In a federated repository, you can create multiple realms.|
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/traditional_deployment/user_registry_consideration/plan_ureg_ov.md:37]:
```suggestion
|Create a realm|Choose this option to create a realm, which is a group of users from one or more user registries that form a coherent group within HCL Portal. Realms facilitate flexible user management with various configuration options. A realm must be mapped to a virtual portal to allow the defined users to log in to the virtual portal. In a federated repository, you can create multiple realms.|
```

**Review by biancaluna — COMMENTED** [review]
Hi @philip-menzies , please see suggested changes.

File: docs/get_started/plan_deployment/traditional_deployment/user_registry_consideration/plan_ureg_ov.md (hunk 1 above)
Comment (philip-menzies) [reply docs/get_started/plan_deployment/traditional_deployment/user_registry_consideration/plan_ureg_ov.md:3]: great

File: docs/get_started/plan_deployment/traditional_deployment/user_registry_consideration/plan_ureg_ov.md (hunk 2 above)
Comment (philip-menzies) [reply docs/get_started/plan_deployment/traditional_deployment/user_registry_consideration/plan_ureg_ov.md:37]: great

**Review by biancaluna — APPROVED** [review]

**Pull request merged**